FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,46 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: self.url) { [weak self] (result) in
+			guard self != nil else { return }
+			switch result {
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			case .success(let (data, httpResponse)):
```

Comment:
The code convention for declaring associated values in enums in this repository is to have the `let` keyword follow `case`:

```suggestion
			case let .success((data, httpResponse)):
```

Reply:
Done 👍🏻. Thank you for the feedback.